etc/_mods/disable_logging_and_monitoring.yaml

```diff
@@ -0,0 +1,65 @@
+kube_prometheus_stack:
+  _install: false
+mongodb:
+  _install: false
+elasticsearch:
+  _install: false
+graylog:
+  _install: false
+fluent_bit:
+  _install: false
+
+cert_manager:
+  prometheus:
+    servicemonitor:
+      enabled: true
+postgresql:
+  metrics:
+    enabled: false
+  primary:
+    sidecars: []
+timescaledb:
+  metrics:
+    enabled: false
+  primary:
+    sidecars: []
+radar_grafana:
+  _install: false
+radar_jdbc_connector:
```

Comment:
Aren't these part of RADAR-Base apps? Why are they being disabled?

Reply:
Ah yes, I mistook it for graylog... updated.